### DISCUSSION TITLE
Shutdown button doesn't work with default gpio_settings.ini

### DISCUSSION THREAD

**Discussion by biohazardxxx** [MiczFlor/RPi-Jukebox-RFID#1356] [Q&A]
### What I did
I installed the raspberry pi with latest buster image and ran the one line installer script.
Connected push buttons to the pins as described in the wiki (no resisters used, directly conneted to GND). All Buttons work fine except the Shurtdown GPIO3 (PIN5).
The button work to "wakeup" the system but not to shutdown. Similar as #1015 
Measuring the voltage it looks also fine. When the buton pressed 0.0v and when not pressed 3.3v.
Checking ~/RPi-Jukebox-RFID/settings/gpio_settings.ini I could see that:
`[Shutdown]
enabled: False`

After changing to True it works!

Is that by purpose? Just wondering

**Comment by s-martin** [comment]
Yes, that's intentional.

I clarified the docs a little: https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/develop/components/gpio_control/README.md
